src/SFA.DAS.AssessorService.Web.Staff/Views/CertificateApprovals/SentForApproval.cshtml

```diff
@@ -0,0 +1,140 @@
+                                                                            {
+                                                                                {
+                                                                                    "Increment", 0
+                                                                                },
+                                                                                { "ApprovalResults", approvalResults },
+                                                                                {"ActionHint","Approved" }
+                                                                            });
+                   
+                    <button type="submit" class="button" role="button">Submit</button>
+                }
+                else
+                {
+                    <div class="info-summary">
+                        <h3 class="heading-medium">No privately funded certificates found for approval</h3>
+                    </div>
+                }
+            </form>
+
+        </div>
+    </div>
+</div>
+@if (Model.SentForApprovalCertificates.HasPreviousPage || Model.SentForApprovalCertificates.HasNextPage)
+{
+    <div class="grid-row">
+        <div class="column-full">
+            <div class="page-navigation">
+
+                @if (Model.SentForApprovalCertificates.HasPreviousPage)
+                {
+                    <a asp-action="SentForApproval"
+                       asp-controller="CertificateApprovals"
+                       asp-route-pageIndex="@(Model.SentForApprovalCertificates.PageIndex - 1)"
+                       class="previous">
+                        <i class="arrow-button icon-left-open">&#xe800;</i>
+                        <span class="description">Previous <span class="hide-mob">page</span></span>
+                        <span class="counter">Page <span class="prevPage">@(Model.SentForApprovalCertificates.PageIndex - 1)</span> of <span class="totalPage">@(Model.SentForApprovalCertificates.TotalPages)</span></span>
+                    </a>
+                }
+
+                @if (Model.SentForApprovalCertificates.HasNextPage)
+                {
+                    <a asp-action="SentForApproval"
+                       asp-controller="CertificateApprovals"
+                       asp-route-pageIndex="@(Model.SentForApprovalCertificates.PageIndex + 1)"
+                       class="next">
+                        <i class="arrow-button icon-right-open">&#xe801;</i>
+                        <span class="description">Next <span class="hide-mob">page</span></span>
+                        <span class="counter">Page <span class="nextPage">@(Model.SentForApprovalCertificates.PageIndex + 1)</span> of <span class="totalPage">@(Model.SentForApprovalCertificates.TotalPages)</span></span>
+                    </a>
```

Comment:
Same as the Reason for Change stuff, this next / previous stuff is repeated (apart from action etc) on lots of pages.  Partial View?

Reply:
Will be inefficient to create a viewmodel of the pagniatedlist just for reusing some markup

Reply:
See above comment

Reply:
next/prev logic moved to _ApprovalPaginationLinks partial view

Reply:
New ticket created https://skillsfundingagency.atlassian.net/browse/ON-1827 to address this as Tech Debt